views/includes/inputs/who-is-filing-content-block.html

```diff
@@ -0,0 +1,32 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    {% include "includes/list/errors.html" %}
+
+    <form method="post">
+      {% set fieldParam = {
+        name: 'who_is_registering',
```

Comment:
I think this could be changed to who_is_filing (as not just registering)

Reply:
Ohh yeah so just update the WhoIsRegisteringKey in the existing model here src/model/who.is.making.filing.model.ts to be more generic?

Reply:
Too much knock on.